### ISSUE TITLE
Terms for dorsalization, ventralization, anterioralization and posterioralization.

### ISSUE THREAD

**Issue by malcolmfisher103** [obophenotype/xenopus-phenotype-ontology#50]
We would like terms made for each of the following PATO qualities for the XAO term 'embryo' (XAO:0000113). We also want to create 'tadpole' terms but this will need to wait on the creation of a non stage 'tadpole' term in the XAO.

|Term ID|Name|
|---|---|
|PATO:0000620|wholly dorsalized|
|PATO:0030005|partially dorsalized|
|PATO:0000636|wholly ventralized|
|PATO:0030007|partially ventralized|
|PATO:0000615|wholly anterioralized|
|PATO:0030004|partially anterioralized|
|PATO:0000630|wholly posterioralized|
|PATO:0030006|partially posterioralized|

**Comment by malcolmfisher103** [comment]
Maybe the tadpole stage equivalents should be broken off into a new ticket so the original request can proceed.

**Comment by matentzn** [comment]
@seger use http://purl.obolibrary.org/obo/upheno/patterns-dev/malformedAnatomicalEntityByType.yaml

**Comment by seger** [comment]
An equivalent ticket for tadpoles has been created - see #98.
